service/matching/physical_task_queue_manager.go

```diff
@@ -325,6 +431,7 @@ func (c *physicalTaskQueueManagerImpl) PollTask(
 
 	task.namespace = c.partitionMgr.ns.Name()
 	task.backlogCountHint = c.backlogMgr.db.getApproximateBacklogCount
+	c.TasksDispatchedInIntervals.trackTasks()
```

Comment:
```suggestion
if pollMetadata.forwardedFrom == "" {
        // only track the original polls, not forwarded ones.
	c.TasksDispatchedInIntervals.trackTasks()
}
```

Reply:
Since this branch is not based off the latest master, I shall be *ignoring* these changes for now. I intend on making a new branch, off the latest master, which shall have all the commits related to this project (Backlog Stats) and shall make it a note to include these changes there. @ShahabT